src/ocnicepost.fd/ocnicepost.F90

```diff
@@ -0,0 +1,577 @@
+program ocnicepost
+
+  use utils_mod, only : debug, logunit, getfield, packarrays, remap, dumpnc
+  use outputvars
+  use netcdf
+
+  implicit none
+
+  character(len= 20) :: fnml, ftype
+  character(len=120) :: wgtsdir
+  character(len=120) :: input_file, wgtsfile, output_file
+
+  ! source grid, tripole 1/4 deg, 40 vertical levels
+  integer, parameter :: nxt = 1440, nyt = 1080, nlevs = 40
+
+  ! destination grids
+  integer, parameter :: ndest = 3
+  integer, parameter, dimension(ndest) :: nxrs = (/1440, 720, 360/)
+  integer, parameter, dimension(ndest) :: nyrs = (/ 721, 361, 181/)
+  character(len=4), dimension(ndest)   :: dstgrds = (/'0p25', '0p5 ', '1p0 '/)
```

Comment:
Can we make this namelist configurable so that the number of and the destination grid is not hard-wired.

Reply:
The number of destination grids is dependent on the source grid in the sense that you don't remap the 1/2 deg tripole to the 1/4deg rectilinear. So for 1/4 deg tripole, you have 3 destination grids, for 1/2 deg tripole there are 2 destination grids and for 1deg tripole there is 1 destination grid. We've never set up a remapping of the 5deg tripole, because there was never any intention of anyone looking at that output.

The dimensions of the destination grids can be set based on whether it is a 1/4 deg, 1/2deg or 1 deg. The grid dimensions of those destination grids are fixed though. They are no more variable than the dimensions of the 1/4deg tripole grid.